app/uk/gov/hmrc/agentuserclientdetails/controllers/ClientListController.scala

```diff
@@ -35,11 +36,21 @@ import scala.util.{Failure, Success}
 class ClientListController @Inject()(
                                            cc: ControllerComponents,
                                            workItemRepo: FriendlyNameWorkItemRepository,
-                                           espConnector: EnrolmentStoreProxyConnector
+                                           espConnector: EnrolmentStoreProxyConnector,
+                                           val env: Environment
                                          )(implicit ec: ExecutionContext)
     extends BackendController(cc) with Logging {
 
+  private val isLocalEnv = {
+    if (env.mode.equals(Mode.Test)) false else env.mode.equals(Mode.Dev)
```

Comment:
hmmm I'm not sure this works... run.mode is Prod in most environments, including from service manager

Reply:
use a config flag instead? set in application.conf and can be set once in app-config-base